TransformationSystem/Service/TransformationManagerHandler.py

```diff
@@ -136,33 +136,35 @@ def export_addTaskForTransformation( self, transName, lfns = [], se = 'Unknown'
     res = database.addTaskForTransformation( transName, lfns = lfns, se = se )
     return self._parseRes( res )
 
-  types_setFileStatusForTransformation = [transTypes, [basestring, dict]]
-  def export_setFileStatusForTransformation( self, transName, dictOfNewFilesStatus, lfns = [], force = False ):
+  def _wasFileInError( self, newStatus, currentStatus ):
```

Comment:
Yes, I know... I did that at some point and reverted to a method such that it can be overloaded in a TS extension. With a function it would not be possible...
If you think this can be hardcoded, I can make it a function and DB then includes it from Client. Anyway the use of it in DB is only temporary while not all clients are updated...